packages/blade-old/src/tokens/theme.web.js

```diff
@@ -1,206 +1,216 @@
 import { colors, fonts, spacings } from '../tokens';
 
+/**
+ * Styled components merges passed themes when multiple theme providers are nested.
```

Comment:
let's elaborate this a bit that it merges when then top level keys are same. update this in the PR description too